lib/mongo/client.rb

```diff
@@ -341,6 +343,19 @@ def with(new_options = Options::Redacted.new)
       end
     end
 
+    # Get the read concern for this client.
+    #
+    # @example Get the client read concern.
+    #   client.read_concern
+    #
+    # @return [ Symbol ] The read concern.
```

Comment:
In collection.rb read_concern is documented as a Hash. Why is it a Symbol here?

Reply:
Ah, this is a mistake left over from when I originally had this method just returning the level rather than the entire read concern. I'll fix it.